From 0b77e8b342756c4b3df91d713d19c2424ce77b6a Mon Sep 17 00:00:00 2001
From: Michael Bestas <mikeioannina@gmail.com>
Date: Wed, 8 Oct 2014 22:45:05 +0200
Subject: [PATCH 036/192] Snap: Add support for additional ISO values

Author: Michael Bestas <mikeioannina@gmail.com>
Date:   Wed Oct 8 22:45:05 2014 +0200

    Snap: Add ISO values for sony devices

    Change-Id: I6d76e2c02ed6222e06aaa59161b3b16e885e6462

Author: codeworkx <codeworkx@cyanogenmod.org>
Date:   Tue Jan 5 19:28:40 2016 +0100

    add additional ISO values

    Change-Id: I373457adfa7c6808a1b0303f2f6b10e0b27a1ecb

Change-Id: I82d11d55b96d8afd90478f08b2a1f0df056924e9
---
 res/values/qcomarrays.xml  | 44 ++++++++++++++++++++++++++++++++++++++
 res/values/qcomstrings.xml | 44 ++++++++++++++++++++++++++++++++++++++
 2 files changed, 88 insertions(+)

diff --git a/res/values/qcomarrays.xml b/res/values/qcomarrays.xml
index 0a2c33b7c..f3b4e1d9e 100755
--- a/res/values/qcomarrays.xml
+++ b/res/values/qcomarrays.xml
@@ -393,22 +393,66 @@
      <string-array name="pref_camera_iso_entries">
          <item>@string/pref_camera_iso_entry_auto</item>
          <item>@string/pref_camera_iso_entry_isodeblur</item>
+         <item>@string/pref_camera_iso_entry_iso50</item>
          <item>@string/pref_camera_iso_entry_iso100</item>
+         <item>@string/pref_camera_iso_entry_iso125</item>
+         <item>@string/pref_camera_iso_entry_iso150</item>
          <item>@string/pref_camera_iso_entry_iso200</item>
+         <item>@string/pref_camera_iso_entry_iso250</item>
+         <item>@string/pref_camera_iso_entry_iso300</item>
+         <item>@string/pref_camera_iso_entry_iso320</item>
+         <item>@string/pref_camera_iso_entry_iso350</item>
          <item>@string/pref_camera_iso_entry_iso400</item>
+         <item>@string/pref_camera_iso_entry_iso450</item>
+         <item>@string/pref_camera_iso_entry_iso500</item>
+         <item>@string/pref_camera_iso_entry_iso600</item>
+         <item>@string/pref_camera_iso_entry_iso640</item>
+         <item>@string/pref_camera_iso_entry_iso700</item>
          <item>@string/pref_camera_iso_entry_iso800</item>
+         <item>@string/pref_camera_iso_entry_iso1000</item>
+         <item>@string/pref_camera_iso_entry_iso1200</item>
+         <item>@string/pref_camera_iso_entry_iso1250</item>
+         <item>@string/pref_camera_iso_entry_iso1500</item>
          <item>@string/pref_camera_iso_entry_iso1600</item>
+         <item>@string/pref_camera_iso_entry_iso2000</item>
+         <item>@string/pref_camera_iso_entry_iso2500</item>
+         <item>@string/pref_camera_iso_entry_iso2700</item>
+         <item>@string/pref_camera_iso_entry_iso3200</item>
+         <item>@string/pref_camera_iso_entry_iso6400</item>
+         <item>@string/pref_camera_iso_entry_iso12800</item>
     </string-array>
 
     <!-- Do not localize entryvalues -->
     <string-array name="pref_camera_iso_entryvalues">
          <item>@string/pref_camera_iso_value_auto</item>
          <item>@string/pref_camera_iso_value_isodeblur</item>
+         <item>@string/pref_camera_iso_value_iso50</item>
          <item>@string/pref_camera_iso_value_iso100</item>
+         <item>@string/pref_camera_iso_value_iso125</item>
+         <item>@string/pref_camera_iso_value_iso150</item>
          <item>@string/pref_camera_iso_value_iso200</item>
+         <item>@string/pref_camera_iso_value_iso250</item>
+         <item>@string/pref_camera_iso_value_iso300</item>
+         <item>@string/pref_camera_iso_value_iso320</item>
+         <item>@string/pref_camera_iso_value_iso350</item>
          <item>@string/pref_camera_iso_value_iso400</item>
+         <item>@string/pref_camera_iso_value_iso450</item>
+         <item>@string/pref_camera_iso_value_iso500</item>
+         <item>@string/pref_camera_iso_value_iso600</item>
+         <item>@string/pref_camera_iso_value_iso640</item>
+         <item>@string/pref_camera_iso_value_iso700</item>
          <item>@string/pref_camera_iso_value_iso800</item>
+         <item>@string/pref_camera_iso_value_iso1000</item>
+         <item>@string/pref_camera_iso_value_iso1200</item>
+         <item>@string/pref_camera_iso_value_iso1250</item>
+         <item>@string/pref_camera_iso_value_iso1500</item>
          <item>@string/pref_camera_iso_value_iso1600</item>
+         <item>@string/pref_camera_iso_value_iso2000</item>
+         <item>@string/pref_camera_iso_value_iso2500</item>
+         <item>@string/pref_camera_iso_value_iso2700</item>
+         <item>@string/pref_camera_iso_value_iso3200</item>
+         <item>@string/pref_camera_iso_value_iso6400</item>
+         <item>@string/pref_camera_iso_value_iso12800</item>
      </string-array>
 
     <!-- Camera Preferences Anti Banding dialog box entries -->
diff --git a/res/values/qcomstrings.xml b/res/values/qcomstrings.xml
index 1d3237b9d..2ca6731eb 100755
--- a/res/values/qcomstrings.xml
+++ b/res/values/qcomstrings.xml
@@ -462,20 +462,64 @@
      <!-- Settings screen, ISO dialog radio button choices -->
      <string name="pref_camera_iso_entry_auto">Auto</string>
      <string name="pref_camera_iso_entry_isodeblur">ISO Auto(HJR)</string>
+     <string name="pref_camera_iso_entry_iso50">ISO50</string>
      <string name="pref_camera_iso_entry_iso100">ISO100</string>
+     <string name="pref_camera_iso_entry_iso125">ISO125</string>
+     <string name="pref_camera_iso_entry_iso150">ISO150</string>
      <string name="pref_camera_iso_entry_iso200">ISO200</string>
+     <string name="pref_camera_iso_entry_iso250">ISO250</string>
+     <string name="pref_camera_iso_entry_iso300">ISO300</string>
+     <string name="pref_camera_iso_entry_iso320">ISO320</string>
+     <string name="pref_camera_iso_entry_iso350">ISO350</string>
      <string name="pref_camera_iso_entry_iso400">ISO400</string>
+     <string name="pref_camera_iso_entry_iso450">ISO450</string>
+     <string name="pref_camera_iso_entry_iso500">ISO500</string>
+     <string name="pref_camera_iso_entry_iso600">ISO600</string>
+     <string name="pref_camera_iso_entry_iso640">ISO640</string>
+     <string name="pref_camera_iso_entry_iso700">ISO700</string>
      <string name="pref_camera_iso_entry_iso800">ISO800</string>
+     <string name="pref_camera_iso_entry_iso1000">ISO1000</string>
+     <string name="pref_camera_iso_entry_iso1200">ISO1200</string>
+     <string name="pref_camera_iso_entry_iso1250">ISO1250</string>
+     <string name="pref_camera_iso_entry_iso1500">ISO1500</string>
      <string name="pref_camera_iso_entry_iso1600">ISO1600</string>
+     <string name="pref_camera_iso_entry_iso2000">ISO2000</string>
+     <string name="pref_camera_iso_entry_iso2500">ISO2500</string>
+     <string name="pref_camera_iso_entry_iso2700">ISO2700</string>
+     <string name="pref_camera_iso_entry_iso3200">ISO3200</string>
+     <string name="pref_camera_iso_entry_iso6400">ISO6400</string>
+     <string name="pref_camera_iso_entry_iso12800">ISO12800</string>
 
      <!-- ISO entry values. Do not translate. -->
      <string name="pref_camera_iso_value_auto">auto</string>
      <string name="pref_camera_iso_value_isodeblur">ISO_HJR</string>
+     <string name="pref_camera_iso_value_iso50">ISO50</string>
      <string name="pref_camera_iso_value_iso100">ISO100</string>
+     <string name="pref_camera_iso_value_iso125">ISO125</string>
+     <string name="pref_camera_iso_value_iso150">ISO150</string>
      <string name="pref_camera_iso_value_iso200">ISO200</string>
+     <string name="pref_camera_iso_value_iso250">ISO250</string>
+     <string name="pref_camera_iso_value_iso300">ISO300</string>
+     <string name="pref_camera_iso_value_iso320">ISO320</string>
+     <string name="pref_camera_iso_value_iso350">ISO350</string>
      <string name="pref_camera_iso_value_iso400">ISO400</string>
+     <string name="pref_camera_iso_value_iso450">ISO450</string>
+     <string name="pref_camera_iso_value_iso500">ISO500</string>
+     <string name="pref_camera_iso_value_iso600">ISO600</string>
+     <string name="pref_camera_iso_value_iso640">ISO640</string>
+     <string name="pref_camera_iso_value_iso700">ISO700</string>
      <string name="pref_camera_iso_value_iso800">ISO800</string>
+     <string name="pref_camera_iso_value_iso1000">ISO1000</string>
+     <string name="pref_camera_iso_value_iso1200">ISO1200</string>
+     <string name="pref_camera_iso_value_iso1250">ISO1250</string>
+     <string name="pref_camera_iso_value_iso1500">ISO1500</string>
      <string name="pref_camera_iso_value_iso1600">ISO1600</string>
+     <string name="pref_camera_iso_value_iso2000">ISO2000</string>
+     <string name="pref_camera_iso_value_iso2500">ISO2500</string>
+     <string name="pref_camera_iso_value_iso2700">ISO2700</string>
+     <string name="pref_camera_iso_value_iso3200">ISO3200</string>
+     <string name="pref_camera_iso_value_iso6400">ISO6400</string>
+     <string name="pref_camera_iso_value_iso12800">ISO12800</string>
 
      <!-- Settings screen, Anti Banding title -->
      <string name="pref_camera_antibanding_title">Anti Banding</string>
-- 
2.17.1

